libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/HttpAccess.java

```diff
@@ -72,18 +50,17 @@
  */
 public class HttpAccess extends BroadcastReceiver {
 
-	public static final String USER_AGENT = "User-Agent";
+    // Timeouts
+    public static final int CONNECT_TIMEOUT = 60;
```

Comment:
We didn't lose the connection/read timeouts that we introduced recently.
We could also add a write timeout.